src/LdapTools/Object/LdapObject.php

```diff
@@ -122,6 +122,9 @@ public function get($attribute)
      */
     public function set($attribute, $value)
     {
+        if($value === '' || $value === null)
+            return $this->reset($attribute);
+
```

Comment:
Could you add some braces to this statement? Following PSR-2 styling for the code: http://www.php-fig.org/psr/psr-2/#5-1-if-elseif-else . Also, I can always add this later if needed, but could you include a spec with this too? It would go in here: https://github.com/ldaptools/ldaptools/blob/master/spec/LdapTools/Object/LdapObjectSpec.php . It would probably look something like:

```
    function it_should_remove_an_attribute_completely_when_calling_set_with_an_empty_string_or_null()
    {
        $this->set('lastName', '');
        $this->has('lastName')->shouldBeEqualTo(false);

        $this->set('lastName', 'foo');
        $this->set('lastName', null);
        $this->has('lastName')->shouldBeEqualTo(false);
    }
```